components/Buttons/Buttons.css

```diff
@@ -0,0 +1,45 @@
+.btn-section{
+    margin: 2rem;
+}
+.primary-btn{
+    padding: 0.25rem 0.5rem;
+    height: 2rem;
+    width: 5rem;
+    font-size: var(--text-size-s);
+    border-radius: var(--border-radius-l);
+}
+.outlined-primary-btn{
+    padding: 0.25rem 0.5rem;
+    height: 2rem;
+    width: 5rem;
+    font-size: var(--text-size-s);
+    border-radius: var(--border-radius-l);
+    border: 2.5px solid var(--primary-color);
+    color: var(--primary-color);
+}
+
+.link-btn{
+    height: 2rem;
+    width: 4rem;
+    background-color: var(--primary-color);
+    border: 2px solid var(--white);
+    color: white;
+    font-size: large;
+    font-weight: 400;
```

Comment:
font weight can also be stored in variables 
height and width are repeated you can create a separate class for them